plugin-test/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,6 @@
+#Mon Oct 03 12:57:02 BST 2016
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+zipStoreBase=GRADLE_USER_HOME
+zipStorePath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-2.13-all.zip
```

Comment:
For some reason, there're two Gradle wrappers in this repository: one at the root level, and second here (at `plugin-test` directory). I think we only need one. So I suggest to delete `plugin-test/gradle/wrapper` directory and `plugin-test/gradlew*` files.